ui/src/app/views/concept-table/component.tsx

```diff
@@ -80,11 +80,28 @@ export class SynonymsObject extends React.Component<{},
   }
 }
 
+interface Props {
+  concepts: Concept[];
+  loading: boolean;
+  onSelectConcepts: Function;
+  nextPage: Function;
+  placeholderValue: string;
+  reactKey: string;
+  selectedConcepts: Concept[];
+}
+
+interface State {
+  first: number;
+  loading: boolean ;
```

Comment:
Can we name this something different to differentiate from the prop `loading`? What are they loading? We should specify